client/dist/styles.css

```diff
@@ -602,10 +602,10 @@
   font-size: 1.4em;
   font-weight: bold;
 }
-
+/*
```

Comment:
Was giving a warning because it was an empty rule. I didn't know what you wanted to do with it so I commented it out.